packages/dids/tests/didmanager.test.js

```diff
@@ -0,0 +1,23 @@
+import {DIDKeyManager} from '../src';
+const didKeyDriver = require('@digitalbazaar/did-method-key').driver();
+
+describe('DID module', () => {
+  it('Create DID', async () => {
+    const keyDoc = {};
+    const {didResolution} = await DIDKeyManager.createDID({});
+    expect(didResolution).toHaveProperty('correlation');
+    expect(didResolution).toHaveProperty('@context');
+    expect(didResolution).toHaveProperty('id');
+    expect(didResolution).toHaveProperty('type', ['DIDResolutionResponse']);
+    expect(didResolution).toHaveProperty('didDocument');
+    expect(didResolution.correlation.length).toBe(0);
+
+    console.log(didResolution);
+    expect(didKeyDriver._keyPairToDidDocument).toBeCalledWith({
```

Comment:
this isnt a valid test, no need to spy that this method gets called. should be able to check that it actually generates and returns a did:key document

Reply:
I had to mock it, that is why i had to check if it was called. I can't actually create a did:key in test env

Reply:
you can, why do you think you cant?